First-Quizz-Page/Foodlist/FoodList.js

```diff
@@ -0,0 +1,76 @@
+import styled from 'styled-components';
+import Image from 'next/image';
+import header_picture from '/public/header-picture.jpeg';
+
+export default function Foodlist() {
+  return (
+    <AppContainer>
+      <Image src={header_picture} alt="wine picture" />
+      <StyledHeader>Was möchtest du Essen?</StyledHeader>
+      <Container>
+        <FoodButton>Seafood</FoodButton>
+        <FoodButton>Fleisch</FoodButton>
+        <FoodButton>Vegetarisch</FoodButton>
+        <FoodButton>Käse</FoodButton>
+        <FoodButton>Dessert</FoodButton>
+        <FoodButton>Pasta</FoodButton>
+      </Container>
+      <BackArrow>&larr;</BackArrow>
+      <StyledFooter>Noch 3 Fragen bis zum perfekten Wein!</StyledFooter>
```

Comment:
From a semantic perspective, I wouldn't put content like this in the footer, better use footer für general app info and make a div here

Reply:
Thanks for the review, makes sense!